adapters/handlers/grpc/v1/tenants.go

```diff
@@ -23,11 +23,12 @@ func (s *Service) tenantsGet(ctx context.Context, principal *models.Principal, r
 	if req.Collection == "" {
 		return nil, fmt.Errorf("missing collection %s", req.Collection)
 	}
+	isConsistent := req.GetConsistencyLevel() == pb.ConsistencyLevel_CONSISTENCY_LEVEL_ALL
 
 	var err error
 	var tenants []*models.Tenant
 	if req.Params == nil {
-		tenants, err = s.schemaManager.GetConsistentTenants(ctx, principal, req.Collection, req.IsConsistent, []string{})
+		tenants, err = s.schemaManager.GetConsistentTenants(ctx, principal, req.Collection, isConsistent, []string{})
```

Comment:
The consistent flag here isn't meant to reflect the consistency level when getting tenants (I.E it isn't ONE, QUORUM or ALL) it's meant to represent whether or not we want to query the leader for the information, or if we are fine with reading the local state on the follower which might eventually consistent.

Reply:
@dirkkul and I had a convo on the Python PR about the origin of the `isConsistent` flag and its relation to the `ConsistencyLevel` [here](https://github.com/weaviate/weaviate-python-client/pull/1031#discussion_r1583414232). Would you say that the concepts of ONE, QUORUM, ALL are no longer applicable to eventual consistency? If so, we should look at a new way to represent this in the clients